client/src/components/pages/members/memberCheckIn/pages/SelfRegisterMember.js

```diff
@@ -0,0 +1,75 @@
+import { bindActionCreators } from 'redux';
```

Comment:
shouldn't there be like a RegisterMemberPage and a RegisterMember component, because this is just like the register member page.